test/act-mapping/karma.config.js

```diff
@@ -0,0 +1,36 @@
+module.exports = function(config) {
+  config.set({
+    basePath: '../../',
+    singleRun: true,
+    autoWatch: false,
+    plugins: [
+      'karma-mocha',
+      'karma-chai',
+      'karma-mocha-reporter',
+      'karma-chrome-launcher',
+      require('./preprocessor')
+    ],
+    frameworks: ['mocha', 'chai'],
+    files: [
+      'axe.js',
+      'test/act-mapping/*.json',
+      {
+        pattern: 'node_modules/act-rules.github.io/test-assets/**/*',
+        included: false,
+        served: true
+      }
+    ],
+    browsers: ['ChromeHeadless'],
```

Comment:
Instead of changing the file path in the runner script we should use a proxy to serve the assets at the correct path

```json
proxies: {
      '/test-assets': '/base/node_modules/act-rules.github.io/test-assets',
    },
```